jOOQ-examples/jOOQ-spring-boot-example/pom.xml

```diff
@@ -11,7 +11,7 @@
 
     <groupId>org.jooq</groupId>
     <artifactId>jooq-spring-boot-example</artifactId>
-    <version>${org.jooq.version}</version>
+    <version>${jooq.version}</version>
```

Comment:
So essentially, once you're done having fun writing 1-2 hello worlds, you'll inevitably need a special version of everything... :)

Oh well.